docs/style.css

```diff
@@ -75,6 +75,9 @@ h2, .subheader {
   background-color: firebrick;
   color: white;
 }
+.firebrick a{
```

Comment:
This should be extracted to our readme template: https://github.com/Appsilon/readme_rmd_template